public/js/main.js

```diff
@@ -59,4 +59,9 @@ $(function() {
 			}
 		});
 	});
+
+	$('.bio-modal').on('shown.bs.modal', function () {
+  		$(this).find('textarea').focus();
+  		console.log('t');
```

Comment:
This is a handy little gulp plugin: https://www.npmjs.com/package/gulp-strip-debug